### ISSUE TITLE
Add primary key in documentation addition result if set

### ISSUE THREAD

**Issue by MarinPostma** [meilisearch/transplant#47]
When performing a documentation with a primary key, we would like the primary_key to be returned in the update result in case of success:

```json
"success":{
    "DocumentsAddition":{
        "nb_documents":1,
        "primaryKey": "smth"
    }
}
```

**Comment by tpayet** [comment]
Thanks for bringing this, we opened meilisearch/specifications#12 which will be resolved for v1.

For v0.21 could we keep the actual behavior in order to minimize changes for the upcoming release of the new engine?

If yes, feel free to re-label this issue :)

**Comment by curquiza** [comment]
Closed in favor of #78
